### ISSUE TITLE
There's no "lang" property in Quasar's type

### ISSUE THREAD

**Issue by Melchyore** [quasarframework/quasar#8552]
**Describe the bug**
Quasar's type is missing "lang" property and probably many others.

**To Reproduce**
1. In a boot file, import Quasar:
```ts
import { Quasar } from 'quasar'

const langIso = Quasar.lang.getLocale()
```

**Expected behavior**
Quasar.lang should not be undefined.

**Screenshots**
[Screenshot](https://cdn.discordapp.com/attachments/806172040804827188/815584247682301972/unknown.png)

**Platform (please complete the following information):**
Quasar Version: 2.0.0-beta.7
@quasar/app Version: 3.0.0-beta.7
Quasar mode:
  [ x] SPA
  [ ] SSR
  [ ] PWA
  [ ] Electron
  [ ] Cordova
  [ ] Capacitor
  [ ] BEX
Tested on:
  [x ] SPA
  [ ] SSR
  [ ] PWA
  [ ] Electron
  [ ] Cordova
  [ ] Capacitor
  [ ] BEX
OS: W7
Node: 14.7.0
NPM: 6.14.7
Yarn: 1.22.4
Browsers: Brave (Based on Chromium)

**Comment by ejez** [comment]
For a quick workaround:
```ts
import { Quasar as Q } from 'quasar'
type QType = typeof Q
interface QuasarType extends QType {
  lang: { getLocale(): string | undefined }
}
const Quasar = Q as QuasarType

const userLocale = Quasar.lang.getLocale()
```

**Comment by IlCallo** [comment]
Seems like this problem affect Qv1 too, but its type were more lax, so you'd get any property except `install` method as `any` type.
Gonna see what I can do about this

**Comment by IlCallo** [comment]
Fix is available in `@quasar/app` v2.2.2
Thanks for reporting!
